flash/vision/classification/data.py

```diff
@@ -524,3 +528,52 @@ def from_filepaths(
             seed=seed,
             **kwargs
         )
+
+
+class ImageClassificationDataViz(ImageClassificationData):
+
+    def show_train_batch(self):
+        self.viz.enabled = True
+        # fetch batch and cache data
+        _ = next(iter(self.train_dataloader()))
+        self.viz.enabled = False
+
+        from typing import List
+
+        import kornia as K
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import torchvision as tv
+        from PIL import Image
+
+        # plot row data
+        rows: int = 4  # chenge later
+        data_raw: List[Image] = self.viz.batches['train']['load_sample']
+        for num, x_data in enumerate(data_raw):
+            img, label = x_data
+            plt.subplot(rows, rows, num + 1)
+            plt.title(label)
+            plt.axis('off')
+            plt.imshow(np.array(img))
+        plt.title('load_sample')
+        plt.show(block=False)
+
+        mean = torch.tensor([0.485, 0.456, 0.406])
+        std = torch.tensor([0.229, 0.224, 0.225])
```

Comment:
Why hardcode these?

Reply:
From Edgar. Might be better to add it within a following PR.